src/backend/bs_worker

```diff
@@ -3258,6 +3258,11 @@ sub dobuild {
     if (($buildinfo->{'reason'} || '') eq "rebuild counter sync") {
       $ret = 0;
     } else {
+      my $statsfile = "$buildroot/.mount/.build.packages/OTHER/_statistics";
+      updatestatsfromfile($stats, $statsfile) if (-e $statsfile);
+      $stats->{'times'}->{'total'}->{'time'} = { 'unit' => 's', '_content' => (time() - $starttime) };
```

Comment:
I think you should keep this in sync with the $ret == 0 case and do this only
`if ($vm =~ /(xen|kvm|zvm|emulator|pvm|openstack)/)`

Also, I would not be surprised if you need that rename() done in the other cases to get the _statistics file in the correct place.